jobs/JRRFS_RUN_MAXTMINRH

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
+
+##########################################################
+# This job...
+# 
+########################################################
+
+date
+export PS4=' $SECONDS + ' 
+set -x
+
+if [ ${cyc} != "07" ] ; then
+    echo -e "\nWARNING: GETMINT only runs at 07z. Exiting.\n"
+    exit 0
+fi
+
+
+#######################################################
+## The following variable could be defined in the
+## submission script (the ecf script), if
+## not they will take the default values which is set
+## for the NCO running enviroment
+########################################################
+export RUN_ENVIR=${RUN_ENVIR:-nco}
```

Comment:
`RUN_ENVIR` is not used in the RRFS workflow any more. Please update your scripts with the latest `dev-sci` branch.